src/Container/ResponseFactoryFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Container;
+
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Zend\Diactoros\Response;
+
+/**
+ * Produces a callable capable of producing a response prototype for use with
+ * services that need to produce a response.
+ */
+class ResponseFactoryFactory
```

Comment:
This class is new in v2, and we did not previously define a `ResponseInterface` service. I've ensured that all factories in zend-expressive and zend-expressive-router that might possibly inject a response instance will pull the service, and, if it's a factory, invoke it to get the response prototype to inject. I will do so with other components as well.